### ISSUE TITLE
Error getting Energy Framework

### ISSUE THREAD

**Issue by DeAnte85** [dylan-jayatilaka/tonto#34]
Hi, long time no speak, I hope all is well!!!

I am having an issue with running Energy Model Calculation B3LYP/6-31G(d,p). I keep receiving error calculating wavefunction... There was an error generating the wavefunction. The program output will be shown for inspection..... The output file is shown below, please help me. Thank you

 Entering Gaussian System, Link 0=C:\G16W\g16.exe
 Input=18087_in_P-1_18087.gjf
 Output=C:\Users\Public\Documents\Xray\Juglone\Dylan - Block\18087_in_P-1_18087.out
 Initial command:
 C:/G16W\l1.exe ".\Gau-10136.inp" "C:\Users\Public\Documents\Xray\Juglone\Dylan - Block\18087_in_P-1_18087.out" /scrdir=".\"
 Entering Link 1 = C:/G16W\l1.exe PID=     15468.
  
 Copyright (c) 1988,1990,1992,1993,1995,1998,2003,2009,2016,
            Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision A.03,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2016.
 
 ******************************************
 Gaussian 16:  EM64W-G16RevA.03 17-Dec-2016
                11-Oct-2022 
 ******************************************
 ---------------------------------------
 #P UB3LYP/6-31G(d,p) 6d 10f NoSymm FChk
 ---------------------------------------
 1/38=1,172=1/1;
 2/12=2,15=1,17=6,18=5,40=1/2;
 3/5=1,6=6,7=101,8=22,11=2,25=1,30=1,74=-5,116=2/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,9=1/99;
 Leave Link    1 at Tue Oct 11 20:11:26 2022, MaxMem=           0 cpu:               0.0 elap:               0.1
 (Enter C:/G16W\l101.exe)
 -------------------
 CrystalExplorer Job
 -------------------
 Symbolic Z-matrix:
 Charge =  0 Multiplicity = 2
 BR                    2.3012   -0.33528   1.88909 
 C                     1.50022   0.53994   3.36177 
 C                     2.03848   0.27501   4.64128 
 C                     0.47902   1.384     3.17731 
 O                     3.0302   -0.59332   4.78567 
 C                     1.47296   0.96223   5.73505 
 H                     2.85199  -0.42713   4.77632 
 H                     0.09966   1.54875   2.17636 
 C                    -0.11318   2.06398   4.26355 
 H                     3.30959  -0.5929    5.72817 
 H                     3.31549  -1.32331   4.19232 
 C                     2.03348   0.75386   7.07226 
 C                     0.39279   1.84255   5.53458 
 O                    -1.06433   2.95375   3.99465 
 H                    -0.94283   2.74225   4.10735 
 O                     2.9975    0.02189   7.27025 
 C                     1.433     1.47243   8.21222 
 C                    -0.23242   2.52668   6.70376 
 H                    -1.27112   2.93653   3.03378 
 N                     1.92909   1.33713   9.49797 
 C                     0.38628   2.32428   8.03313 
 O                    -1.23004   3.20013   6.59515 
 N                     2.82796   0.51158   9.74914 
 N                    -0.12072   2.97099   9.15891 
 N                     3.61168  -0.14272   10.22485 
 N                    -1.0739    3.77134   9.04997 
 N                    -1.89449   4.51276   9.19931 
 
 ITRead=  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0
 ITRead=  0  0  0  0  0  0  0
 MicOpt= -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1
 MicOpt= -1 -1 -1 -1 -1 -1 -1
 NAtoms=     27 NQM=       27 NQMF=       0 NMMI=      0 NMMIF=      0
                NMic=       0 NMicF=      0.
                    Isotopes and Nuclear Properties:
 (Nuclear quadrupole moments (NQMom) in fm**2, nuclear magnetic moments (NMagM)
  in nuclear magnetons)

  Atom         1           2           3           4           5           6           7           8           9          10
 IAtWgt=          79          12          12          12          16          12           1           1          12           1
 AtmWgt=  78.9183361  12.0000000  12.0000000  12.0000000  15.9949146  12.0000000   1.0078250   1.0078250  12.0000000   1.0078250
 NucSpn=           3           0           0           0           0           0           1           1           0           1
 AtZEff=   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000
 NQMom=   31.3000000   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000
 NMagM=    2.1063990   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000   2.7928460   2.7928460   0.0000000   2.7928460
 AtZNuc=  35.0000000   6.0000000   6.0000000   6.0000000   8.0000000   6.0000000   1.0000000   1.0000000   6.0000000   1.0000000

  Atom        11          12          13          14          15          16          17          18          19          20
 IAtWgt=           1          12          12          16           1          16          12          12           1          14
 AtmWgt=   1.0078250  12.0000000  12.0000000  15.9949146   1.0078250  15.9949146  12.0000000  12.0000000   1.0078250  14.0030740
 NucSpn=           1           0           0           0           1           0           0           0           1           2
 AtZEff=   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000
 NQMom=    0.0000000   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000   2.0440000
 NMagM=    2.7928460   0.0000000   0.0000000   0.0000000   2.7928460   0.0000000   0.0000000   0.0000000   2.7928460   0.4037610
 AtZNuc=   1.0000000   6.0000000   6.0000000   8.0000000   1.0000000   8.0000000   6.0000000   6.0000000   1.0000000   7.0000000

  Atom        21          22          23          24          25          26          27
 IAtWgt=          12          16          14          14          14          14          14
 AtmWgt=  12.0000000  15.9949146  14.0030740  14.0030740  14.0030740  14.0030740  14.0030740
 NucSpn=           0           0           2           2           2           2           2
 AtZEff=   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000   0.0000000
 NQMom=    0.0000000   0.0000000   2.0440000   2.0440000   2.0440000   2.0440000   2.0440000
 NMagM=    0.0000000   0.0000000   0.4037610   0.4037610   0.4037610   0.4037610   0.4037610
 AtZNuc=   6.0000000   8.0000000   7.0000000   7.0000000   7.0000000   7.0000000   7.0000000
 Leave Link  101 at Tue Oct 11 20:11:26 2022, MaxMem=   104857600 cpu:               0.0 elap:               0.2
 (Enter C:/G16W\l202.exe)
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1         35           0        2.301204   -0.335277    1.889086
      2          6           0        1.500215    0.539938    3.361766
      3          6           0        2.038480    0.275008    4.641283
      4          6           0        0.479015    1.384004    3.177305
      5          8           0        3.030196   -0.593322    4.785671
      6          6           0        1.472962    0.962232    5.735049
      7          1           0        2.851991   -0.427131    4.776325
      8          1           0        0.099663    1.548751    2.176360
      9          6           0       -0.113180    2.063975    4.263551
     10          1           0        3.309588   -0.592899    5.728173
     11          1           0        3.315493   -1.323308    4.192324
     12          6           0        2.033482    0.753863    7.072256
     13          6           0        0.392787    1.842552    5.534580
     14          8           0       -1.064332    2.953746    3.994648
     15          1           0       -0.942831    2.742252    4.107345
     16          8           0        2.997502    0.021885    7.270254
     17          6           0        1.433000    1.472432    8.212218
     18          6           0       -0.232420    2.526675    6.703764
     19          1           0       -1.271122    2.936534    3.033776
     20          7           0        1.929092    1.337130    9.497966
     21          6           0        0.386280    2.324280    8.033129
     22          8           0       -1.230042    3.200131    6.595150
     23          7           0        2.827959    0.511576    9.749143
     24          7           0       -0.120717    2.970985    9.158911
     25          7           0        3.611676   -0.142720   10.224852
     26          7           0       -1.073901    3.771341    9.049974
     27          7           0       -1.894493    4.512764    9.199305
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  Br   0.000000
     2  C    1.891130   0.000000
     3  C    2.831265   1.413181   0.000000
     4  C    2.817057   1.337656   2.409364   0.000000
     5  O    2.998036   2.377525   1.326026   3.606270   0.000000
     6  C    4.142577   2.410715   1.410110   2.776306   2.397091
     7  H    2.940740   2.182544   1.083068   3.386457   0.243852
     8  H    2.911850   2.093905   3.384860   1.083024   4.470459
     9  C    4.150150   2.395610   2.823601   1.411732   4.149054
    10  H    3.977662   3.187010   1.884226   4.292693   0.983041
    11  H    2.703681   2.730704   2.094502   4.050359   0.983023
    12  C    5.303127   3.754714   2.477692   4.240765   2.834928
    13  C    4.655601   2.764836   2.442024   2.403008   3.667461
    14  O    5.155375   3.578255   4.149845   2.348203   5.474721
    15  H    4.991555   3.372617   3.906482   2.175192   5.231742
    16  O    5.437772   4.217407   2.809855   4.995035   2.559824
    17  C    6.633520   4.939732   3.814709   5.125257   4.308085
    18  C    6.147450   4.256536   3.805367   3.774619   4.904908
    19  H    4.977607   3.678527   4.541064   2.343913   5.833554
    20  N    7.799389   6.202612   4.972669   6.485035   5.210067
    21  C    6.963436   5.123120   4.293472   4.946893   5.103790
    22  O    6.864098   4.998570   4.801789   4.230941   5.984491
    23  N    7.923076   6.523978   5.173923   7.033329   5.088984
    24  N    8.345503   6.491864   5.686773   6.217540   6.462025
    25  N    8.440344   7.212922   5.815985   7.862084   5.488702
    26  N    8.918182   7.030203   6.430227   6.526804   7.353786
    27  N    9.723512   7.834810   7.362229   7.189379   8.354929
                    6          7          8          9         10
     6  C    0.000000
     7  H    2.179725   0.000000
     8  H    3.859303   4.270742   0.000000
     9  C    2.427960   3.906506   2.160353   0.000000
    10  H    2.406590   1.069060   5.244584   4.573777   0.000000
    11  H    3.316416   1.165772   4.759663   4.820224   1.700695
    12  C    1.464829   2.708506   5.323655   3.770063   2.291020
    13  C    1.407810   3.431340   3.383767   1.385837   3.804816
    14  O    3.665104   5.232488   2.575862   1.329922   5.891942
    15  H    3.413788   4.989308   2.497987   1.082950   5.642108
    16  O    2.359095   2.538202   6.056116   4.784008   1.689192
    17  C    2.529480   4.174594   6.181844   4.281655   3.735994
    18  C    2.508828   4.685465   4.643706   2.486554   4.819683
    19  H    4.327209   5.599177   2.130764   1.901193   6.379622
    20  N    3.808956   5.124281   7.549669   5.665534   4.454450
    21  C    2.883960   4.925116   5.914840   3.811422   4.729533
    22  O    3.613059   5.755706   4.901110   2.823931   5.978877
    23  N    4.260523   5.060698   8.115810   6.414979   4.197622
    24  N    4.277585   6.292157   7.129329   4.978682   6.019819
    25  N    5.094443   5.508580   8.942792   7.367575   4.529243
    26  N    5.036500   7.162672   7.318724   5.171840   7.021106
    27  N    5.995590   8.154411   7.879327   5.790623   8.074590
                   11         12         13         14         15
    11  H    0.000000
    12  C    3.775209   0.000000
    13  C    4.512929   2.498314   0.000000
    14  O    6.124960   4.889541   2.393604   0.000000
    15  H    5.888066   4.647884   2.151821   0.268687   0.000000
    16  O    3.374065   1.226511   3.621040   5.985307   5.738520
    17  C    5.245902   1.475274   2.896339   5.120435   4.909888
    18  C    5.806667   2.900510   1.491945   2.865969   2.700472
    19  H    6.365966   5.656305   3.196781   0.983023   1.139329
    20  N    6.095070   2.497032   4.280667   6.469972   6.267464
    21  C    6.052848   2.470377   2.544573   4.337030   4.165695
    22  O    6.848136   4.106391   2.366728   2.617399   2.545843
    23  N    5.872199   2.802788   5.046197   7.363989   7.143155
    24  N    7.410498   3.729656   3.830512   5.249792   5.123135
    25  N    6.154097   3.637777   6.025048   8.382636   8.154170
    26  N    8.295721   4.761551   4.269590   5.121023   5.050325
    27  N    9.288356   5.838036   5.078572   5.496195   5.474343
                   16         17         18         19         20
    16  O    0.000000
    17  C    2.332177   0.000000
    18  C    4.126412   2.482033   0.000000
    19  H    6.683119   6.022635   3.836105   0.000000
    20  N    2.798941   1.384761   3.727562   7.388181   0.000000
    21  C    3.563914   1.361375   1.480191   5.302393   2.345311
    22  O    5.331898   3.562534   1.208549   3.571352   4.677311
    23  N    2.532476   2.287202   4.764544   8.232803   1.246030
    24  N    4.689084   2.357101   2.497526   6.232327   2.643134
    25  N    3.022243   3.377283   5.856689   9.221448   2.355718
    26  N    5.813961   3.503052   2.786034   6.077042   3.891535
    27  N    6.915262   4.614127   3.596493   6.394282   4.979322
                   21         22         23         24         25
    21  C    0.000000
    22  O    2.333966   0.000000
    23  N    3.491761   5.800291   0.000000
    24  N    1.393793   2.802852   3.884811   0.000000
    25  N    4.614425   6.913149   1.126328   4.976158   0.000000
    26  N    2.293488   2.525237   5.132194   1.249399   6.217302
    27  N    3.369177   2.991007   6.213967   2.350531   7.283075
                   26         27
    26  N    0.000000
    27  N    1.115966   0.000000
 Small interatomic distances encountered:      7     5    15    14
 Problem with the distance matrix.
 Error termination via Lnk1e in C:/G16W\l202.exe at Tue Oct 11 20:11:26 2022.
 Elapsed time:       0 days  0 hours  0 minutes  0.3 seconds.
 File lengths (MBytes):  RWF=      6 Int=      0 D2E=      0 Chk=      1 Scr=      1

**Comment by peterspackman** [comment]
Since no-one else has commented here, I'll give it a try.

You have two atoms which are too close together, with indices 5 & 7:

```
O 3.0302 -0.59332 4.78567
H 2.85199 -0.42713 4.77632
```
Then you have another two atoms with indices 14 & 15, which are too close together:
```
O -1.06433 2.95375 3.99465
H -0.94283 2.74225 4.10735
```

That's a pretty good indicator there's something wrong with the structure. Is it disordered? I don't know. But there's not really a sensible answer you're going to get with two atoms being so close together, the energies will be extremely large.

**Comment by DeAnte85** [comment]
Thank you so much for replying. At first we thought that it was disordered but our crystallographer informed that it was co-crystals at 80:20 %. I have the pure derivative of one of them with the intermolecular energies. But for the co-crystal, the calculation kept failing. I got it to work but I advised my PI that i don't believe it's correct but she says that she loves it.

On Nov 1, 2022 10:10 PM, Peter Spackman ***@***.***> wrote:

External Email: Use Caution


Since no-one else has commented here, I'll give it a try.

You have two atoms which are too close together, with indices 5 & 7:

O 3.0302 -0.59332 4.78567
H 2.85199 -0.42713 4.77632


Then you have another two atoms with indices 14 & 15, which are too close together:

O -1.06433 2.95375 3.99465
H -0.94283 2.74225 4.10735


That's a pretty good indicator there's something wrong with the structure. Is it disordered? I don't know. But there's not really a sensible answer you're going to get with two atoms being so close together, the energies will be extremely large.
